File: abc/abc205/abc205_b.fs
// https://atcoder.jp/contests/abc205/submissions/23917931

let N = stdin.ReadLine() |> int

stdin.ReadLine().Split()
|> Array.map int
|> Array.sort
|> Array.mapi (fun i x -> i + 1 = x)
|> Array.filter id
|> fun x -> if x.Length = N then "Yes" else "No"
|> stdout.WriteLine

File: abc/abc205/abc205_b.cs
// https://atcoder.jp/contests/abc205/submissions/23917120

using System;
using System.Linq;

class Program
{
    static void Main()
    {
        var N = int.Parse(Console.ReadLine());
        var A = Console.ReadLine().Split().Select(int.Parse).ToArray();
        Array.Sort(A);

        for (int i = 0; i < N; i++)
        {
            if (i + 1 != A[i])
            {
                Console.WriteLine("No");
                return;
            }
        }
        Console.WriteLine("Yes");
    }
}